content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -171,23 +207,43 @@ As a best practice, add the `.env` file to `.gitignore` to prevent `git` from mo
 
 ## 5. Push the Prisma schema to the database
 
-The `prisma/schema.prisma` file contains a pre-defined schema for the app. It defines the `Post`, `Profile`, and `User` models. Run `prisma db push` to create the database tables and their relationships based on the models and relationships defined in `prisma/schema.prisma`.
+In the Netlify deployment example repository, the `prisma/schema.prisma` file contains a pre-defined schema for the app. It defines the `Post`, `Profile`, and `User` models.
+
+### Steps
 
 ```terminal
 npx prisma db push
 ```
 
+### Result
+
+`prisma db push` creates the database tables and their relationships based on the models and relationships defined in `prisma/schema.prisma`.
```

Comment:
```suggestion
`prisma db push` creates the database tables and their foreign keys based on the models and relationships defined in `prisma/schema.prisma`.
```

Reply:
(The first "relationships" could also stay if only the second one is changed to "relations")